content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -739,6 +739,106 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Reflection Paths: Enums and Tuples
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
+
+Bevy's "reflection paths" enable navigating Rust values using a simple (and dynamic) string syntax. **Bevy 0.10** expands this system by adding support for tuples and enums in reflect paths:
+
+```rust
+#[derive(Reflect)]
+struct MyStruct {
+  data: Data,
+  some_tuple: (u32, u32),
+}
+
+#[derive(Reflect)]
+struct Data {
+  Foo(u32, u32),
+  Bar(bool)
+}
+
+let x = MyStruct {
+  data: Data::Foo(123),
+  some_tuple: (10, 20),
+};
+
+assert_eq!(*x.path::<u32>("data.1").unwrap(), 123);
+assert_eq!(*x.path::<u32>("some_tuple.0").unwrap(), 10);
+```
+
+## Pre-Parsed Reflection Paths
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
+
+Reflection paths enable a lot of interesting and dynamic editor scenarios, but they do have a downside: calling `path()` requires parsing strings every time. To solve this problem we added [`ParsedPath`], which enables pre-parsing paths and then reusing those results on each access:
+
+```rust
+let parsed_path = ParsedPath::parse("foo.bar[0]").unwrap();
+let element = parsed_path.element::<usize>(&some_value);
+```
+
+Much more suitable for repeated access, such as doing the same lookup every frame!
+
+## `ReflectFromReflect`
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
+
+When using Bevy's Rust reflection system, we sometimes end up in a scenario where we have a "dynamic reflect value" representing a certain type `MyType` (but under the hood, it is not that type). Such scenarios happen when we call `Reflect::clone_value`, use the reflection deserializers, or create the dynamic value ourselves. Unfortunately, we can't just call `MyType::from_reflect` as we do not have knowledge of the concrete `MyType` at runtime.
```

Comment:
Nit: I think this could be a little clearer. Maybe something like?
```suggestion
When using Bevy's Rust reflection system, we sometimes end up in a scenario where we have a "dynamic reflect value" representing a certain type `MyType` (even though under the hood, it isn't really that type). Such scenarios happen when we call `Reflect::clone_value`, use the reflection deserializers, or create the dynamic value ourselves. Unfortunately, we can't just call `MyType::from_reflect` as we do not have knowledge of the concrete `MyType` at runtime.
```